Projects/Team1/Semeniuk/Feature_with_project_class_Animal_Fish/StarProject/Fish.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace StarProject
+{
+    class Fish : Animals
+    {
+        private string _kind;
+        public string kind
```

Comment:
you can use short variant:  public string Kind { get; set; } - automatic property https://metanit.com/sharp/tutorial/3.4.php

property name should start from uppercase